src/hooks/use-toast.ts

```
@@ -1,106 +1,109 @@
-  id: string
-  title?: React.ReactNode
-  description?: React.ReactNode
-  action?: ToastActionElement
-}
+  id: string;
+  title?: React.ReactNode;
+  description?: React.ReactNode;
+  action?: ToastActionElement;
+};
 
 const actionTypes = {
```

ESLint

[error] 'actionTypes' is assigned a value but only used as a type.

(@typescript-eslint/no-unused-vars)